list2.py

```diff
@@ -18,17 +18,25 @@
 
 
 def remove_adjacent(nums):
-    """Your code goes here.  Edit this docstring."""
-    return
+    newList = []
+    for num in nums:
+        if not newList or newList[-1] != num:
+            newList.append(num)
+    return newList
 
 
 # E. Given two lists sorted in increasing order, create and return a merged
 # list of all the elements in sorted order. You may modify the passed in lists.
 # The solution should work in "linear" time, making a single pass of both lists.
 # Hint: Don't use `sort` or `sorted` -- they are not linear time.
 def linear_merge(list1, list2):
```

Comment:
The code is very nice. As frankly, annoying, as it may be, please consider line breaks after all the colons.